types/react/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React 15.6
+// Type definitions for React 16.0
```

Comment:
That's actually not necessary, but if you do want to maintain multiple versions of react in parallel, you need to copy *all* of the old definitions to `v15` (incl. `tsconfig.json`, `tslint.json`, and `test`), not just `index.d.ts`.
`README.md` is ignored by types-publisher, so this might be a good time to move it into a comment instead.